docs/en/events.rst

```diff
@@ -79,3 +79,11 @@ The events you can listen for are as follows:
   in implementing projects or extending packages. See
   `Package Intersphinx, provided by the TYPO3 Documentation Team <https://github.com/TYPO3-Documentation/intersphinx>`__
   for an example implementation.
+- ``PreDocumentRenderEvent``: This event is called in the `DocumentNodeRenderer`
```

Comment:
Shouldn't you be using double backticks? Since this isn't markdown?

Reply:
As our default textrole is "code" the result is exactly the same: https://github.com/doctrine/rst-parser/pull/234

Also GiThub renders it like code as well. The only advantage of double backticks then is that you can use single backticks within without escaping them